Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.cpp

```diff
@@ -586,6 +586,31 @@ namespace AZ
 #endif
         }
 
+        void CommandList::UpdateBottomLevelAccelerationStructure(const RHI::RayTracingBlas& rayTracingBlas)
+        {
+#ifdef AZ_DX12_DXR_SUPPORT
+            const RayTracingBlas& dx12RayTracingBlas = static_cast<const RayTracingBlas&>(rayTracingBlas);
+            const RayTracingBlas::BlasBuffers& blasBuffers = dx12RayTracingBlas.GetBuffers();
+
+            // create the BLAS
+            D3D12_BUILD_RAYTRACING_ACCELERATION_STRUCTURE_DESC blasDesc = {};
+            blasDesc.Inputs = dx12RayTracingBlas.GetInputs();
+            blasDesc.Inputs.Flags = D3D12_RAYTRACING_ACCELERATION_STRUCTURE_BUILD_FLAG_PERFORM_UPDATE;
```

Comment:
You overwrite the `Flags` with the `PERFORM_UPDATE` flag here. According to the DXR spec (https://microsoft.github.io/DirectX-Specs/d3d/Raytracing.html#buildraytracingaccelerationstructure), the other flag states must match the ones in the Build command: 
> The other flags selections, aside from ALLOW_UPDATE and PERFORM_UPDATE, must match the flags in the source acceleration structure.

You will have to add the flag, and not replace the flags with it

Reply:
I have patched this into a pending PR now.

Reply:
Where?